module/platform/darwin/HardwareIO/src/darwin/UART.cpp

```diff
@@ -133,17 +133,17 @@ namespace Darwin {
         // We flush our buffer, just in case there was anything random in it
         tcflush(fd, TCIFLUSH);
 
-        while ((bytesWritten != (int) count) && (reconnects < 3)) {
+        while ((bytesWritten != static_cast<int>(count)) && (reconnects < 3)) {
             bytesWritten = write(fd, buf, count);
 
-            if (bytesWritten < (int) count) {
+            if (bytesWritten < static_cast<int>(count)) {
                 reconnect();
                 reconnects++;
             }
         }
 
         if (reconnects > 0) {
-            std::cout << "Bytes Written: " << (int) bytesWritten << " Reconnects: " << (int) reconnects << "\r"
+            std::cout << "Bytes Written: " << bytesWritten << " Reconnects: " << static_cast<int>(reconnects) << "\r"
```

Comment:
```suggestion
            std::cout << "Bytes Written: " << bytesWritten << " Reconnects: " << int(reconnects) << "\r"
```

Reply:
I added this suggestion, but I am curious - what's wrong with `static_cast`? I like it :(

Reply:
So many letters! It constantly makes things split one-liners into multi-liners!